### DISCUSSION TITLE
Can not disable or hide timeline feature in Vuex

### DISCUSSION THREAD

**Discussion by ferrykranenburgcw** [vuejs/devtools-v6#1655] [Q&A] [answered]
### Version
6.0.0-beta.21

### Browser and OS info
Windows 10, Chrome

### Steps to reproduce
I have disabled the timeline option in Vuex (plugin options) but it still shows the timeline.

### What is expected?
The timelink is hidden or can be removed (or resized or even dragged to another position)

### What is actually happening?
Vuex timeline is still there

**Comment by Akryum** [comment, marked as answer]
You can remove the layer in the interface

**Reply by ferrykranenburgcw** [reply]
I did, but I also needed to restart the browser because settings where not applied correctly. Thanks.
